libs/libradcli/Makefile

```diff
@@ -8,23 +8,24 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libradcli
-PKG_VERSION:=1.2.5
+PKG_VERSION:=1.2.10
 PKG_RELEASE:=1
 
 PKG_SOURCE:=radcli-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://github.com/radcli/radcli/releases/download/$(PKG_VERSION)/
-PKG_HASH:=be7551bcf0c210b8efffe4d2952d61feb8591edc922660910488bfaab040e82c
+PKG_SOURCE_URL:=https://github.com/radcli/radcli/releases/download/radcli_1_2_10
```

Comment:
Hmmm, actually that was an issue upstream. You can replace it back with `PKG_VERSION`